docs/04-rollout/02-learning-goals.adoc

```diff
@@ -22,8 +28,11 @@
   * Durch DevOps wird der Aufbau der Teams anders. Es müssen neben Entwicklung auch Betrieb stärker betrachtet werden. Neben dem Provisioning hat das auch Auswirkungen auf Conti- nuous Delivery (siehe Kapitel „Continuous Delivery“).
 
 .Was sollen die Teilnehmer kennen?
-  * Grundlegende Konzept von moderner Infrastruktur wie IaaS, PaaS und Virtualisierung
-  * Konzepte von Deployment-Werkzeuge wie Chef, Puppet, Ansible oder Salt
+  * Grundlegende Konzept von moderner Infrastruktur wie IaaS
+    (virtuelle Maschinen auf Abruf), PaaS (Deployment-Umgebung auf
+    Abruf) und Virtualisierung
+  * Konzepte von Deployment-Werkzeuge wie Chef, Puppet, Ansible, Salt
```

Comment:
```suggestion
  * Konzepte von Deployment-Werkzeugen wie Chef, Puppet, Ansible, Salt
```

Reply:
Ich tendiere sehr stark dazu diesen Punkt zu streichen und einen LINK auf Cloud-Infra zu setzen, weil das Kernthema des CLOUDINFRA Moduls ist. Es sollte maximal als COULD in FLEX drin bleiben (cc @sippsack @TimRiemer)